startup/STM32F4xx/STM32F446xx.s

```diff
@@ -0,0 +1,406 @@
+.include "startup_common.s"
```

Comment:
You can now delete the STM32F446xE.s file since we now have this one. These files are included based on their mapped device name, so the old E file will not be used after merging these changes.